tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -331,7 +331,7 @@ sub_port_interfaces:
   skip:
     reason: "Unsupported platform or asic"
     conditions:
-      - "is_multi_asic==True or asic_gen not in ['td2', 'spc1', 'spc2', 'spc3']"
+      - "is_multi_asic==True or asic_gen not in ['td2', 'spc1', 'spc2', 'spc3'] and asic_type not in ['barefoot']"
```

Comment:
If asic is non barefoot but TD2, it will be skipped. Is it expected?
Should change AND to OR?

Reply:
Hi @ZhaohuiS. If ASIC is TD2 and not barefoot, it won't be skipped. For example:
```
>>> is_multi_asic=False
>>> asic_gen='td2'
>>> asic_type='test'
>>> is_multi_asic==True or asic_gen not in ['td2', 'spc1', 'spc2', 'spc3'] and asic_type not in ['barefoot']
False
>>> 
```
Thanks!

Reply:
Not really) If it's barefoot, whatever asic_gen, it will be passed. Also, if it isn't barefoot, but asic_gen is 'td2', it will be passed as well. In another way, the test case is skipped.